instrumentation/volley/library/build.gradle.kts

```diff
@@ -0,0 +1,72 @@
+import java.time.Duration
+
+plugins {
+    id("com.android.library")
+    id("otel.android-library-conventions")
+    id("otel.errorprone-conventions")
```

Comment:
Both plugins, `otel.errorprone-conventions` and `com.android.library`, are added by `otel.android-library-conventions`, so it's not needed to explicitly declare them here ;)